packages/hooks/README.md

```diff
@@ -0,0 +1,34 @@
+# WP-JS-Hooks
+
+A lightweight & efficient EventManager for JavaScript in WordPress.
+
+
+### API Usage
+API functions can be called via the global `wp.hooks` like this `wp.hooks.addAction()`, etc.
+
+* `addAction( 'namespace.identifier', callback, priority )`
+* `addFilter( 'namespace.identifier', callback, priority )`
+* `removeAction( 'namespace.identifier', callback )`
+* `removeFilter( 'namespace.identifier',  callback )`
+* `removeAllActions(  'namespace.identifier' )`
+* `removeAllFilters(  'namespace.identifier' )`
+* `doAction( 'namespace.identifier', arg1, arg2, moreArgs, finalArg )`
+* `applyFilters( 'namespace.identifier', content )`
+* `doingAction( 'namespace.identifier' )`
+* `doingFilter( 'namespace.identifier' )`
+* `didAction( 'namespace.identifier' )`
+* `didFilter( 'namespace.identifier' )`
+* `hasAction( 'namespace.identifier' )`
+* `hasFilter( 'namespace.identifier' )`
```

Comment:
Yes, action name should come first if that is the way we want to go.  However, if it's required, maybe we should just make it a separate argument.  See discussion starting at https://core.trac.wordpress.org/ticket/21170#comment:129.